From 0087694b7663c51a37beb533eda87bbb791d91d5 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Fri, 25 Jul 2025 11:33:20 +0200
Subject: [PATCH 899/954] feat(drivers/scmi): add define for SCMI protocol 17

Add define to identify the SCMI protocol 17, the "Voltage domain
management protocol".

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 61a408897728880267ed4f1d28deef92b2b2744c feat(drivers/scmi): add define for SCMI protocol 17]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I668458b4622870d9533c49997c1ed4d324224dab
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/477697
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
---
 include/drivers/scmi.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/drivers/scmi.h b/include/drivers/scmi.h
index ac5dc38717..f8acc6a28e 100644
--- a/include/drivers/scmi.h
+++ b/include/drivers/scmi.h
@@ -12,6 +12,7 @@
 #define SCMI_PROTOCOL_ID_CLOCK			0x14U
 #define SCMI_PROTOCOL_ID_SENSOR			0x15U
 #define SCMI_PROTOCOL_ID_RESET_DOMAIN		0x16U
+#define SCMI_PROTOCOL_ID_VOLTAGE_DOMAIN		0x17U
 
 /* SCMI error codes reported to agent through server-to-agent messages */
 #define SCMI_SUCCESS			0
-- 
2.53.0

